src/components/gcds-checkbox/gcds-checkbox.css

```diff
@@ -0,0 +1,118 @@
+:host {
+  .gcds-checkbox-wrapper {
+    max-width: 90%;
+    font-family: var(--gcds-font-families-body), sans-serif;
+    font-weight: var(--gcds-font-weights-regular);
+    font-size: var(--gcds-font-sizes-paragraph);
+    line-height: var(--gcds-line-heights-paragraph);
+    color: var(--gcds-checkbox-default-text);
+    margin: 0;
+    padding: 0;
+    border: 0;
+    transition: color ease-in-out 0.15s;
+
+    position: relative;
+    margin-bottom: 20px;
+    margin-top: 28px;
+
+    &:focus-within {
+      color: var(--gcds-checkbox-focus);
+    }
+
+    &.gcds-disabled {
+      color: var(--gcds-checkbox-disabled-text);
+
+      input[type=checkbox]+gcds-label {
+        color: var(--gcds-checkbox-disabled-text);
+        &:before {
+          border-color: var(--gcds-checkbox-disabled-border);
+          background-color: var(--gcds-color-grayscale-100);
+          cursor: not-allowed;
+        }
+        &:after {
+          cursor: not-allowed;
+        }
+      }
+    }
+
+    &.gcds-error:not(:focus-within) {
+      color: var(--gcds-checkbox-destructive);
+
+      input[type=checkbox]+gcds-label {
+        &:before {
+          border-color: var(--gcds-checkbox-destructive);
+        }
+        &:after {
+          color: var(--gcds-checkbox-destructive);
+        }
+      }
+    }
+
+    input[type=checkbox] {
+      height: 40px;
+      left: 0;
+      opacity: 0;
+      position: absolute;
+      top: 0;
+      width: 40px;
+
+      &:checked+gcds-label:after {
+        opacity: 1;
+      }
+
+      &+gcds-label {
+        width: fit-content;
+        &:before {
+          border: 2px solid var(--gcds-color-grayscale-1000);
+          content: "";
+          height: 40px;
+          left: 0;
+          top: -8px;
+          position: absolute;
+          width: 40px;
+          cursor: pointer;
+          border-radius: var(--gcds-spacing-50);
+        }
+        &:after {
+          content: "";
+          border: 5px solid;
+          border-left: 0;
+          border-top: 0;
+          height: 20px;
+          color: var(--gcds-color-grayscale-1000);
+          left: 13px;
+          opacity: 0;
+          position: absolute;
+          top: -2px;
+          transform: rotate(40deg);
+          transition: opacity .2s ease-in-out;
+          width: 12px;
+          cursor: pointer;
```

Comment:
Can we group shared stylings like `cursor: pointer;` together for the before and after?